### ISSUE TITLE
Move all HTML out of GridField components

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-framework#8623]
## Affected Version

4.3.x-dev

## Description

Plenty of our GridFieldComponents render HTML in the PHP class definitions.

We should remove these and move them into `$this->renderWith(self::class)` and a SilverStripe template, in order to separate concerns of PHP and HTML.

**Comment by maxime-rainville** [comment]
I agree that it's a bad practice, but if we're going to be shifting our efforts to a react/graphql gridfield is there any point putting effort on this?

I quick search seems to indicate there's about 5 files with HTML snipets under `src/Forms/GridField`.

**Comment by robbieaverill** [comment]
Probably not, but PHP GridField will be supported for the live of SS4 at least, and someone else might want an easy issue to jump into helping out with =)
